src/structures/interfaces/TextBasedChannel.js

```diff
@@ -67,6 +68,19 @@ class TextBasedChannel {
    * @property {UserResolvable} [reply] User to reply to (prefixes the message with a mention, except in DMs)
    */
 
+  /**
+   * Options provided to control parsing of mentions by Discord
+   * @typedef {Object} MessageMentionOptions
+   * @property {MessageMentionTypes[]} [parse] Types of mentions to be parsed
+   * @property {UserResolvable[]} [users] Snowflakes of Users to be parsed as mentions
+   * @property {RoleResolvable[]} [roles] Snowflakes of Roles to be parsed as mentions
```

Comment:
theres an inconsistency from the typings here, the typings state these should be `Snowflake`s, but here it is `UserResolvable`s